interfaces/builtin/fwupd.go

```diff
@@ -72,6 +72,8 @@ const fwupdPermanentSlotAppArmor = `
 
   # Allow write access for efi firmware updater
   /boot/efi/{,**/} r,
+  /boot/efi/EFI/boot/fwupd*.efi* rw,
+  /boot/efi/EFI/boot/fw/** rw,
```

Comment:
I'm ok with the change based on the description, but can you add a terse supporting comment above the boot/ and the ubunmtu/ lines? Something like:
```
  # Allow reading all directories in /boot/efi
  /boot/efi/{,**/} r,
  # allow access to fwupd* and fw/ under boot/ to allow [your text here]
  /boot/efi/EFI/boot/fwupd*.efi* rw,
  /boot/efi/EFI/boot/fw/** rw,
  # allow access to fwupd* and fw/ under ubuntu/ to allow [your text here]
  /boot/efi/EFI/ubuntu/fwupd*.efi* rw,
  /boot/efi/EFI/ubuntu/fw/** rw,
```

Reply:
@jdstrand Updated. Thanks for a comment.